ui/components/MesheryPerformanceComponent.js

```diff
@@ -117,8 +116,15 @@ class MesheryPerformanceComponent extends React.Component {
 
   handleChange = (name) => (event) => {
     if (name === 'url' && event.target.value !== '') {
+      const compulsoryProtocolValidUrlPattern = new RegExp('^(https?:\\/\\/)' // compulsory protocol
```

Comment:
This is an improvement. Thank you, @shubhamgupta2956. At some point, Meshery will support generating load for other protocols like gRPC, TCP and potentially NATS.

Reply:
While we are updating this expression, let's go ahead and include the following protocols as valid:

`nats://`
`tcp://`

Do you mind making this update, @shubhamgupta2956?

Reply:
Yeah sure. I will update the PR in a moment.